doc/ir-semantics.text

```diff
@@ -125,74 +105,50 @@ Update (partial) function, returns an H:
   eval P H E se          = simple-eval P H E se
   eval P H E primop(se*) = 〚primop〛(v*)
     where (v := simple-eval P H E se)*
-  eval P H E [se*]       = [v*]
-    where (v := simple-eval P H E se)*
   eval P H E x[se]       = vₙ
```

Comment:
Currently it has to be a simple expression, which is consistent with the fact that primitive arguments must be simple expressions.